test/Microsoft.Extensions.Logging.Test/ConsoleLoggerTest.cs

```diff
@@ -63,16 +63,19 @@ public void LogsWhenMessageIsNotProvided()
             var sink = t.Item2;
             var exception = new InvalidOperationException("Invalid value");
 
+            ((TestConsole)t.Item1.Console).OnFlush = () =>
```

Comment:
Technically... though the semaphore dispatch is slower than the queue. Could add some `ManualResetEvent`s?